SteamPrefill/Settings/AppConfig.cs

```diff
@@ -31,12 +31,7 @@ public static bool VerboseLogs
         /// Downloaded manifests, as well as other metadata, are saved into this directory to speedup future prefill runs.
         /// All data in here should be able to be deleted safely.
         /// </summary>
-        public static readonly string CacheDir = CacheDirUtils.GetCacheDirBaseDirectories("SteamPrefill", CacheDirVersion);
-
-        /// <summary>
-        /// Increment when there is a breaking change made to the files in the cache directory
-        /// </summary>
-        private const string CacheDirVersion = "v1";
+        public static readonly string TempDir = CacheDirUtils.GetCacheDirBaseDirectories("SteamPrefill", "v1");
```

Comment:
In theory the Utils and function names here could be changed aswell?

Reply:
Got it taken care of